### ISSUE TITLE
Axis Capital Group now in Jakarta – For All Your Construction Equipment Needs

### ISSUE THREAD

**Issue by brettrwang** [AxisCapitalEquipment/Axis-Capital-Group-now-in-Jakarta#1]
<p style="text-align: justify;">With the fast pace of development not just in Asia but throughout the Middle-East, Europe and America, the need for reliable construction equipment has grown. Project managers requiring heavy equipment are always on the look-out for excellent suppliers of equipment that can deliver the best products and service. Axis Capital Group could be the final destination for many of these project managers.</p>
<p style="text-align: justify;">A company located in Singapore, <a href="http://axiscapitalgrp.com/"><span style="color: windowtext;">Axis Capital Group now in Jakarta</span></a> sells and leases out construction capital equipment, ranging from cranes to excavators and other similar or related light and heavy construction equipment. It is a multi-line dealership that can provide dependable service for its clients&rsquo; construction project equipment needs.</p>
<p style="text-align: justify;">With its strategic location in Singapore, which is a bustling port in Southeast Asia, Axis Capital Group assures its clients of efficient delivery of its top-quality products. The company&rsquo;s corporate structure also allows it to operate with great flexibility and with low overheads, making their prices highly competitive in Asia as well as globally.&nbsp;</p>
<p style="text-align: justify;">Axis Capital Group can provide a wide selection of new and used machine units on sale or rental basis. The equipment delivered is assured to be dependable and cost-effective for the prospective owner or user, especially coming from an asset management company that guarantees delivery of full service to its valued clients.</p>
<p style="text-align: justify;">Axis Capital Group carries the following impressive line of reputable equipment manufacturers globally recognized as leaders in the industry:</p>
<ul style="text-align: justify;">
<li>American Crane</li>
<li>Badger Crane</li>
<li>Bobcat</li>
<li>Bomag</li>
<li>Bucyrus-Erie</li>
<li>Case</li>
<li>Caterpillar</li>
<li>Clark Michigan</li>
<li>Dresser</li>
<li>Dynapac</li>
<li>Euclid</li>
<li>FMC</li>
<li>Grove Crane</li>
<li>Ingersoll-Rand</li>
</ul>
<p style="text-align: justify;"></p>
<p style="text-align: justify;">No need to look somewhere else when you can have the best products as well as the excellent service that goes with them. Whether you operate in Asia or any part of the world, you can depend on having a reliable partner for your construction equipment need in Axis Capital Group.</p>

**Comment by bleuennhurst** [comment]
You just envy companies like Axis which now located in Jakarta whose business it is to sell and rent out such equipment that run our industries. We don’t call them prime-movers for nothing. Most of us can only dream of collecting tiny scale-models.

**Comment by jairocliffe** [comment]
Yes, most of us men experienced playing with cans, matchboxes and shoe boxes turned into toy heavy-equipment when we were boys. We even had ships with tiny stick-cranes that actually operated using threads and bent needles to lift cargo. Not many of us, however, got to handle real machines as adults. We know how hard that can be, especially if you work in the desert. But such equipment – or most of them do have air-conditioning and high-tech accessories now. That’s a tribute to the innovations and creativity of machine manufacturers. How I would spend a day, if I had the money, renting out a bulldozer or a crane and playing with it just for fun. Big toys for big boys -- that would be a neat experience!

**Comment by yuriyvovk** [comment]
Those equipment are truly fascinating although many people take them for granted. Where would all the tall buildings and infrastructure around us be without these machines?

**Comment by aidannguyen** [comment]
In the real world, big toys are not much fun although many of the operators probably love their work. They are unsung heroes in reality. Their skill, diligence and endurance can match those of others, such as soldiers, pilots and athletes.

**Comment by tanyav1ck** [comment]
Efficiency is also required in handling such machines. They cost money to operate and each minute, whether idle or not, means cost for the owner or renter. And unless your equipment is in tip-top condition, the project will not go according to schedule and within budget. Axis must know that as an expert provider of such products; otherwise, it would not be where it is now.
